cli/Valet/CommandLine.php

```diff
@@ -25,7 +25,7 @@ function quietly($command)
      */
     function quietlyAsUser($command)
     {
-        $this->quietly('sudo -u '.user().' '.$command.' > /dev/null 2>&1');
+        $this->quietly('sudo '.$command.' > /dev/null 2>&1');
```

Comment:
The method signature does not make sense anymore.

Reply:
Why?

Reply:
Well as the method signature says: quietlyAsUser you just deleted the code that executes the code as a user, therefore, is the signature no longer valid for its functionality.

Reply:
Ahh, yeah